mosaic/get_global_area.c

```diff
@@ -0,0 +1,57 @@
+/***********************************************************************
```

Comment:
I used `git mv` to move read_mosaic.c to get_global_area.c so that Github knows which files to compare, but seems like it's not working here.